scripts/ipnet.js

```diff
@@ -0,0 +1,261 @@
+#!/usr/node/bin/node
+
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2021 Joyent, Inc.
+ */
+
+/*
+ * netmask value indexed by prefix length
+ * Generated with:
+ * var prefix = Array.apply(null, Array(32)).map(function (_, i) {
+ *     return (((0x1ffffffff << (32 - i)) - 0xffffffff) >>> 0);
+ * });
+ */
+var netmask = [
+    0,           // 0
+    2147483648,  // 1
```

Comment:
Does JS not accept hex constants (0x8000000)?  If it DOES, it'll be far more readable to have:

0x80000000,
0xC0000000,
0xE0000000,
0xF0000000,
0xF8000000,
....
0xFFFFFFFC,
0xFFFFFFFE,
0xFFFFFFFF

Reply:
It does, and I use them in a bunch of places, so I don’t know why I didn’t just think of that in the first place.